desktop/hardened-desktop/admin-controls/index.md

```diff
@@ -0,0 +1,57 @@
+---
+description: admin controls for desktop
+keywords: admin controls, rootless, docker desktop, hardened desktop
+title: What is Admin Controls?
+--- 
+>Note
+>
+>Admin Controls is currently in [Early Access](../../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
+
+Admin Controls is a feature that gives administrators the ability to enforce certain Docker Desktop settings for their organization. 
+
+With a few lines of JSON, administrators are able to enforce preferences for HTTP proxies, network settings, and the Docker Engine configuration. This saves significant time and cost in securing developer workflows.
```

Comment:
"admins are able to configure settings for Docker Desktop features such as proxies, network settings..."

Reply:
can we figure out a way to work Enhanced Container Isolation in here ? We're conveying that it's simple to configure admin settings and that it will thereby save time and cost. It would be good to also convey - 'and hey....when you enable Enhanced Container Isolation as part of this, it ensures that any configurations you create cannot be modified'